Makefile

```diff
@@ -56,6 +56,7 @@ test-js-dist: start-support
 	@yarn turbo run test:dist
 test-python:
 	@python3 -m pytest
+	@python3 -m coverage xml -o .coverage.xml
```

Comment:
So I didn't want the coverage creation to be in the `docker/test_python.sh` script, so this is the alternative. I can move coverage creation to that script as an alternative if we want.